### PR TITLE
Rename env vars and properties

### PR DESCRIPTION
**Pull request by jeesmon**
Signed-off-by: Jeesmon Jacob <jjacob@us.ibm.com>

### CODE REVIEW

File: dicom-dimse-proxy/src/main/resources/application.properties [hunk 1]
```diff
@@ -12,19 +12,16 @@ dimse.nats.reply.timeoutSeconds=60
...
-dimse.proxy.host=0.0.0.0
-dimse.proxy.port=11112
-dimse.proxy.aet=DIMSE
-dimse.proxy.device=${dimse.proxy.aet}
-dimse.proxy.actor=PROXY
+dimse.host=0.0.0.0
+dimse.port=11112
+dimse.ae=DIMSE
+dimse.device=${dimse.ae}
+dimse.actor=SCU
```
Comment (rduggan-ibm) [inline dicom-dimse-proxy/src/main/resources/application.properties:19]: The default should be SCP here.  The proxy is a projection of the remote SCP.

File: dicom-dimse-proxy/src/test/resources/application.properties [hunk 2]
```diff
@@ -12,19 +12,16 @@ dimse.nats.traceConnection=false
...
-%test.dimse.proxy.host=0.0.0.0
-%test.dimse.proxy.port=11116
-%test.dimse.proxy.aet=DIMSE
-%test.dimse.proxy.device=${dimse.proxy.aet}
-%test.dimse.proxy.actor=PROXY
+%test.dimse.host=0.0.0.0
+%test.dimse.port=11116
+%test.dimse.ae=DIMSE
+%test.dimse.device=${dimse.ae}
+%test.dimse.actor=SCU
```
Comment (rduggan-ibm) [inline dicom-dimse-proxy/src/test/resources/application.properties:19]: Same as above.  SCP

File: dicom-dimse-service/src/main/resources/application.properties [hunk 3]
```diff
@@ -35,12 +35,11 @@ dimse.nats.traceConnection=false
...
-dimse.calling.aet=LOCAL
+dimse.target.host=localhost
+dimse.target.port=${dimse.port}
+dimse.target.ae=${dimse.ae}
+dimse.target.device=${dimse.device}
 
 dimse.config.path=${CONFIG_PATH:/etc/config/dimse}
 
-dimse.proxy.actor=SERVER
+dimse.actor=SCP
```
Comment (rduggan-ibm) [inline dicom-dimse-service/src/main/resources/application.properties:45]: This should be SCU.  The proxy will act as a SCU to the ingestion code.  This one is a little tricky conceptually as the ingestion is embedded directly.

**Review by rduggan-ibm — COMMENTED** [review]
Thanks for reaching out out-of-band.  I see now that you coded this up from a NATS behaviour whereas I was looking at it from a projection behaviour.  We will get together in a bit and hash this out.  See what makes the most sense.

**Comment by jeesmon** [comment]
@rduggan-ibm pushed updates based on the discussion. Please review when you get a chance. Thanks.

**Review by rduggan-ibm — APPROVED** [review]

**Pull request merged**